SwiftGen.xcodeproj/project.pbxproj

```diff
@@ -556,7 +556,7 @@
 					"$(inherited)",
 					"$(SWIFTGEN_OTHER_LDFLAGS)",
 				);
-				OTHER_SWIFT_FLAGS = "$(inherited) $(SWIFTGEN_OTHER_SWIFT_FLAGS)";
+				OTHER_SWIFT_FLAGS = "$(inherited)";
```

Comment:
Probably better to completely remove the setting if it's gonna only inherit and add nothing (select the setting at the target level and hit delete key)